app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/MessageController.kt

```diff
@@ -72,5 +72,10 @@ object MessageController {
         }
         realm?.let(block) ?: RealmDatabase.mailboxContent().writeBlocking(block)
     }
+
+    fun updateMessage(uid: String, realm: MutableRealm? = null, onUpdate: (message: Message) -> Unit) {
```

Comment:
It is only called by `fetchDraft`.
And the realm is never null. You should make it an extension of MutableRealm like `fetchDraft`.